chromium_src/chrome/browser/profiles/profile.cc

```diff
@@ -68,3 +69,15 @@ bool Profile::OTRProfileID::IsSearchBackupResults() const {
   return base::StartsWith(profile_id_, kSearchBackupResultsOTRProfileIDPrefix,
                           base::CompareCase::SENSITIVE);
 }
+
+// This is to avoid a circular dep on chrome/browser in the Factory for
+// the brave_origin keyed service.
+namespace brave_origin {
+policy::PolicyService* GetPolicyServiceFromProfile(Profile* profile) {
+  policy::PolicyService* policy_service = nullptr;
+  if (auto* connector = profile->GetProfilePolicyConnector()) {
+    policy_service = connector->policy_service();
```

Comment:
nit: you could `return connector->policy_service()` here and `return nullptr` at the end of the function